docs/docs/concepts/structured_output.mdx

```diff
@@ -0,0 +1,120 @@
+# Structured output
```

Comment:
We have some conceptual material here:

https://python.langchain.com/v0.1/docs/modules/model_io/chat/structured_output/
https://python.langchain.com/v0.1/docs/use_cases/extraction/